src/Services/ConvertToLearnosityService.php

```diff
@@ -170,14 +209,11 @@ private function convertQtiContentPackagesInDirectory($sourceDirectory, $relativ
             foreach ($itemResources as $resource) {
                 $itemCount++;
                 $totalItemCount++;
-                $resourceHref    = $resource['href'];
+                $resourceHref = $resource['href'];
                 $relatedResource = $resource['resource'];
                 $assessmentItemContents = file_get_contents($currentDir . '/' . $resourceHref);
-                $itemReference   = $this->getItemReferenceFromResource(
-                    $relatedResource,
-                    $this->useMetadataIdentifier,
-                    $this->useResourceIdentifier,
-                    $this->useFileNameAsIdentifier
+                $itemReference = $this->getItemReferenceFromResource(
+                    $relatedResource, $this->useMetadataIdentifier, $this->useResourceIdentifier, $this->useFileNameAsIdentifier
```

Comment:
Formatting: this will break PSR-2. Arguments in a multi-line method call should each be on a separate line, with correct indentation.

Reply:
Done